stdlib/public/Darwin/Accelerate/vDSP_Conversion.swift

```diff
@@ -0,0 +1,1305 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Accelerate
+
+extension vDSP {
+    
+    // MARK: Integer to floating-point conversion
+    
+    /// Converts an array of unsigned 8-bit integers to single-precision floating-point values.
+    ///
+    /// - Parameter source: Source vector.
+    /// - Parameter destination: Destination vector.
+    @inline(__always)
+    @available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+    public static func convert<U, V>(_ source: U,
+                                     to destination: inout V)
```

Comment:
@airspeedswift @moiseev I don't love this naming when it appears in use:
```
let a: [UInt8]
let b: [Float]
vDSP.convert(a, to: b)
```
This function doesn't convert a to b; it converts UInt8 to Float. Thoughts?

Reply:
Agreed. Something like `convertElements(from: a, to: b)` would probably be appropriate. The type doesn't appear which is a bit unsettling, but the fact that `b` is `inout` probably makes that OK. If OTOH it were returning an array, the type should definitely appear.

Reply:
Or `of:`

Reply:
I like `convertElements(of:to:)`, and I've updated my code accordingly. Thanks!